src/components/AddEvent/FormsUI/Button.js

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@mui/material";
+import { useFormikContext } from "formik";
+
+const ButtonWrapper = ({ children, ...otherProps }) => {
```

Comment:
I think this component is useless and the name is not so suggestive. You can add the submit function in the Button from AddEvent js. 
![image](https://user-images.githubusercontent.com/26420219/159506059-106ae5af-f626-4519-ae5a-061f2dc3dbdc.png)
 check those formik props..submit function is included there.
 Read https://formik.org/docs/api/formik